packages/block-library/src/verse/index.js

```diff
@@ -23,8 +23,7 @@ export const settings = {
 	icon,
 	example: {
 		attributes: {
-			// translators: Sample content for the Verse block. Can be replaced with a locale-appropriate work.
```

Comment:
Can you preserve the `translators` note? Possibly even add that "\n" indicates line breaks?

Reply:
Whoops, sorry. Might have been a bad rebase.